testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -610,7 +612,11 @@ def test_replication_with_session_cookie(setup_client_syncgateway_test):
 
     # Sanity test docs
     ls_docs = client.add_docs(url=ls_url, db=ls_db, number=100, id_prefix="ls_doc", channels=["ABC"])
+    assert len(ls_docs) == 100
```

Comment:
can you change to:

```
num_docs_pushed = 100
assert len(ls_docs) == num_docs_pushed
```

basically name (and therefore document) the expected result, which will make it easier to understand when it fails.  Names above are just an example .. might be something different.